.travis.yml

```diff
@@ -60,15 +61,18 @@ jobs:
         - $UPGRADE_PIP
         - pip install restructuredtext_lint
       script: make rst-lint
+    -
+      name: "Try to build documentation"
+      install:
+        - $UPGRADE_PIP
+        - pip install . sphinx-click
+      script: python setup.py build_sphinx
+
     - stage: "macOS - Unit tests"
       language: generic
       name: "Python 2.7"
       env: PYTHON_VERSION=2.7
-      install:
-        - git clone git://github.com/astropy/ci-helpers.git
-        - source ci-helpers/travis/setup_conda.sh
-        - pip install -e .
-      script: $MAIN_CMD $SETUP_CMD
+      install: pip install . mdbenchmark -v
```

Comment:
This build runs now on ubuntu and *not* osx. Could this be part of the problem? You sed the PYTHON_VERSION environment variable but the global used version is 3.6 set at the top of the file.

Reply:
Wow, that makes sense. Solving month-old merge conflicts is a pain. Thanks for the heads up!